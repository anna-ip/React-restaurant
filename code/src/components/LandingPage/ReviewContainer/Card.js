import React from 'react'

const Card = ({ quote }) => {
  return (
    <>
      <p>{quote.name}</p>
      <p>{quote.text}</p>
    </>
  )
}

export default Card
