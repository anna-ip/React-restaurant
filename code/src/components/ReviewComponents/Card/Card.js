import React from 'react'
import styles from './Card.module.scss'

const Card = ({ quote }) => {
  return (
    <div className={styles.container}>
      <h4>{quote.name}</h4>
      <p>{quote.text}</p>
    </div>
  )
}

export default Card
