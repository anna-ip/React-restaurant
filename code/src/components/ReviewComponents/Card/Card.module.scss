@import '../../../styles/_variables.scss';

.container {
  width: 75%;
  @media (min-width: $layout-breakpoint-small) {
    padding: 1.5rem 0.5rem;
  }

  h4 {
    color: #1c6e8e;
    text-transform: capitalize;
    font-size: 1.1rem;
  }
  .qoutes {
    color: #fff;
    font-size: 2.2rem;
    height: 2rem;
  }
}
