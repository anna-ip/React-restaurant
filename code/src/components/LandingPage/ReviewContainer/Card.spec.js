import React from 'react'
import renderer from 'react-test-renderer'
import Card from './Card'

test('<Card/> renders correctly', () => {
  const tree = renderer.create(<Card quote="" />).toJSON()
  expect(tree).toMatchSnapshot()
})

//? test for the content or props
