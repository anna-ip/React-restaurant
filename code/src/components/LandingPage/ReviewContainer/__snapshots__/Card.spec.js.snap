// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card/> renders correctly 1`] = `
<div
  className="container"
>
  <h4 />
  <p />
</div>
`;
